Source/Libs/KLibc/lib.a/iob_data.a

```diff
@@ -132,7 +135,7 @@ _iob: fdb 0 _ptr
  fcb 0 _save
  fdb 0 _bufsiz
 
- endsect
```

Comment:
can we leave it as `endsect`?

Reply:
Yes.  Since `lwasm` is fine with either; I should have left it alone.

Reply:
I've pushed an update to the PR with this change removed.